src/bsp/platform/stm32fx_nucleo_wifi/xi_bsp_io_net_stm32fx_nucleo_wifi.c

```diff
@@ -50,12 +51,39 @@ xi_bsp_io_net_state_t xi_bsp_io_net_create_socket( xi_bsp_socket_t* xi_socket )
     return XI_BSP_IO_NET_STATE_OK;
 }
 
+static xi_bsp_io_net_state_t xi_bsp_io_net_configure_tls( const char* host )
+{
+    WiFi_Status_t status          = WiFi_MODULE_SUCCESS;
+    posix_time_t current_datetime = xi_bsp_time_sntp_getseconds_posix();
+    uint8_t* tls_mode             = ( uint8_t* )"o"; //["m"utual || "o"ne-way]
+    char* tls_cert                = GLOBALSIGN_ROOT_CERT;
+
+    xi_bsp_debug_format( "Trusted Root CA Certificate:\r\n%s", tls_cert );
+    status =
+        wifi_socket_client_security( tls_mode, ( uint8_t* )tls_cert, NULL, NULL,
+                                     ( uint8_t* )host, ( uint32_t )current_datetime );
+
+    if ( status != WiFi_MODULE_SUCCESS )
+    {
```

Comment:
we had a standard of doing ( Wifi_Fi_MODULE_SUCCESS != status ) just to be safe and not accidentally assign to the status variable instead of comparing against it.